docs/Getting-Started/Create-bootable-media/bootable-usb-using-macos.md

```diff
@@ -0,0 +1 @@
+Users creating installation media from Windows or MacOS can use Rufus. Rufus is an open source startup disk creator capable of writing ISO images to portable storage media.
```

Comment:
Etcher is cross-platform (Windows, Mac and Linux) which is why we usually recommend it for non Pop users.
Popsicle being an AppImage also means that it is available for all (most) Linux distros.

NOTE: this should be changed in the Preparation sub-header in the previous section as well.